application/src/main/resources/locale/blueprint/horizons/description.csv

```diff
@@ -1,176 +1,176 @@
-blueprint.horizons.description.sub_surface_displacement_munitions,Refills Sub Surface Displacement ammunition for a single weapon. Basic synthesis provides half of a weapon''s ammo capacity. Standard and premium synthesis offers a full restock and increased damage until the weapon is reloaded.,Refills Sub Surface Displacement ammunition for a single weapon. Basic synthesis provides half of a weapon''s ammo capacity. Standard and premium synthesis offers a full restock and increased damage until the weapon is reloaded.,"Nachfüllungsverschiebung Munition für eine einzelne Waffe. Die grundlegende Synthese liefert die Hälfte der Munitionskapazität einer Waffe. Die Standard- und Premium -Synthese bietet einen vollständigen Auffüllung und einen erhöhten Schaden, bis die Waffe neu geladen wird.",Recaza la munición de desplazamiento de la superficie subterránea para un solo arma. La síntesis básica proporciona la mitad de la capacidad de munición de un arma. La síntesis estándar y premium ofrece un replanteamiento completo y un mayor daño hasta que el arma se vuelva a cargar.,Recharge des munitions de déplacement sous surface pour une seule arme. La synthèse de base fournit la moitié de la capacité des munitions d''une arme. La synthèse standard et premium offre un réapprovisionnement complet et des dégâts accrus jusqu''à ce que l''arme soit rechargée.,Reabastece a munição de deslocamento da superfície para uma única arma. A síntese básica fornece metade da capacidade de munição de uma arma. A síntese padrão e premium oferece um reabastecimento completo e um aumento de danos até que a arma seja recarregada.,Базовый синтез даёт половину боезапаса орудия. Вытесняющие боеприпасы для добычи глубинных залежей используются ВЫТЕСНЯЮЩИМИ РАКЕТАМИ ДЛЯ ДОБЫЧИ ГЛУБИННЫХ ЗАЛЕЖЕЙ.
-blueprint.horizons.description.thrusters,Thrusters,Thrusters,Triebwerke,Propulsores,Propulseurs,Propulsores,Маневровые двигатели
-blueprint.horizons.description.torpedo_pylon,Torpedo Pylon,Torpedo Pylon,Torpedo Pylon,Pilón de torpedo,Pylône de torpille,Pilão de torpedo,Торпедная установка
-blueprint.horizons.description.caustic_sinks,Refills Caustic sinks for a single module. Basic synthesis provides a full restock of the launcher''s supplies.,Refills Caustic sinks for a single module. Basic synthesis provides a full restock of the launcher''s supplies.,,,,,
-blueprint.horizons.description.caustic_sink_launcher,Caustic Sink Launcher,Caustic Sink Launcher,,,,,
+messageKey,,en,de,es,fr,pt,ru,
```

Comment:
there is an additional comma added to every single line indicating an added language. can you remove these?